jaeger/cmd/install.go

```diff
@@ -190,6 +196,29 @@ func render(w io.Writer, valuesOverrides map[string]interface{}, registry string
 		}
 	}
 
-	_, err = w.Write(buf.Bytes())
-	return err
+	if format == "json" {
+		reader := yamlDecoder.NewYAMLReader(bufio.NewReaderSize(&buf, 4096))
+		for {
+			manifest, err := reader.Read()
+			if err != nil {
+				if errors.Is(err, io.EOF) {
+					break
+				}
+				return err
+			}
+			bytes, err := yaml.YAMLToJSON(manifest)
+			if err != nil {
+				return err
+			}
+			_, err = w.Write(append(bytes, '\n'))
+			if err != nil {
+				return err
+			}
+		}
+		return nil
+	} else if format == "yaml" {
+		_, err = w.Write(buf.Bytes())
+		return err
+	}
+	return fmt.Errorf("unsupported format %s", format)
```

Comment:
This is ripe for moving into a library given it's repeated almost verbatim so many times.